### ISSUE TITLE
Referer Bug

### ISSUE THREAD

**Issue by mrochaaa** [jingyu9575/multithreaded-download-manager#80]
Sometimes when the  plugin capture an new download, the referer field is filed with an extension url like (moz-extension://etc...) and no real referer, then plugin ask to fill the field with an  acceptable referer, I can't start download only if I delete this  wrong url, this bug is not constant, if I close the window and try the same link again, referer is filled perfect.

**Comment by jingyu9575** [comment]
It now clears the field if the value is not acceptable. I'm not sure what is the reason, although I suspect there may be conflicts from other extensions. Do you have the URL, so the source extension can be checked?

`moz-extension://` is not allowed as the referrer for privacy reasons. Websites are not supposed to know you have a specific extension installed.

**Comment by mrochaaa** [comment]
Right! When the issue happen again I can copy the wrong URL. Thanks
